src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -1459,46 +1473,120 @@ def _configure_security_policy_storage_params(arg_ctx):
                    options_list=['--dest-mi'],
                    help='Name of the managed instance to restore managed database to. '
                    'This can be same managed instance, or another managed instance on same subscription. '
-                   'When not specified it defaults to source managed instance.')
+                   'When not specified for PITR it defaults to source managed instance. '
+                   'Required for LTR.')
 
         c.argument('target_resource_group_name',
                    options_list=['--dest-resource-group'],
                    help='Name of the resource group of the managed instance to restore managed database to. '
-                   'When not specified it defaults to source resource group.')
+                   'When not specified for PITR it defaults to source resource group. '
+                   'Required for LTR.')
```

Comment:
Having a different set of required/non-required parameters is a code smell :(

Reply:
Would it be better to split into 2 different functions then?

Reply:
One command should not have 2 completely different set of parameters, instead it should just be 2 commands. (Please note the difference between this case, where you have 1 group of required parameters for mode X and another group of required parameters for mode Y, versus the list scenario where you have a bunch of optional parameters and then we are able to intelligently choose the correct SDK method based on the parameters you specified).

Reply:
moved restore under ltr-backup